sdk/tests/util/test_stream_merge.py

```diff
@@ -0,0 +1,30 @@
+import asyncio
+from collections import defaultdict
+
+import pytest
+from aiostream import stream
+from opentelemetry import trace
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.trace import get_current_span
+
+
+@pytest.mark.skip(reason="Currently preventing us from automatically generating spans on context objects automatically")
```

Comment:
This is a PITA, any thoughts on what we could do here would be nice.

This means that we can not separately track open telemetry contexts from merged streams. I know this must be possible, but this feels like an open telemetry bug